README.md

```diff
@@ -1 +1,31 @@
 # ResetPasswordBundle
+
+Worrying about how to deal with users that can't remember their password? We've got you covered! This bundle provides a secure out of the box solution to allow users to reset their forgotten passwords.
+
+## Requirements
+
+Most Symfony application's using Symfony 4.4 or newer should already have all the required dependencies to use this bundle, and the optional traits that are included with it. 
+
+If your application isn't a traditional Symfony web application, see the [detailed requirements](https://github.com/SymfonyCasts/reset-password-bundle/wiki/Dependencies) in our wiki.
+
+## Installation
+
+@TODO - To be added upon release
+
+## Configuration
+
+We highly recommend using `bin/console make:reset-password` to generate and configure the required application files that the bundle needs to function as intended. 
+
+Symfony's Flex creates the bundle's `config/packages/reset_password.yaml` config file. The `reset_request_repository` param is populated with a fake request repository value which must be changed before use. `bin/console make:reset-password` has you covered there too.
+
+Checkout the [configuration reference](https://github.com/SymfonyCasts/reset-password-bundle/wiki/Confguration-Reference) in the wiki for more detailed configuration information.
+
+Symfony's [Mailer DSN](https://symfony.com/doc/current/mailer.html) must be properly configured to send emails after a reset request has been made.
```

Comment:
Yes... but that's outside of the scope if THIS bundle. This bundle has nothing to do with your controllers or sending emails. MakerBundle may need a note about this at the end of the command (MAYBE), but probably not here.

Reply:
removed from here.. added card on the project board (maker wish list) to refer back to